design/ceph/object/store.md

```diff
@@ -60,6 +60,8 @@ The pools are the backing data store for the object store and are created with s
 
 If `preservePoolsOnDelete` is set to 'true' the pools used to support the object store will remain when the object store will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
 
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. And for this option to work, `preservePoolsOnDelete` must be set to 'true'. Now the object stores can consume same pool isolated with different namespaces.
```

Comment:
What rados namespaces will be created in the pool? Currently we have several metadata pools for an object store. Will rados namespaces be created in place of each of those pools?

Reply:
There will be one pool IMO, In the same pool Rook creates a namespace for RGW AFAIR. Please refer @cbodley comment https://github.com/rook/rook/issues/10805#issuecomment-1340013429

Reply:
Please capture the proposal in this design doc, so we don't have to refer back to that issue. I'm not clear now what the proposal is. We need to be clear exactly which pool(s) are created, which CR(s) own them, and which RADOS namespaces are used by an object store.